muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/reducers/index.ts

```diff
@@ -24,9 +23,8 @@ export interface StateType {
   locales: LocaleListType,
   status: StatusType,
   title: string,
-  
+  credentials: CredentialsType,
```

Comment:
You should add a `credentials?: CredentialsType`, making it not optional, means it's available all the time, and it is not available for most of the app.